pkg/controller/.import-restrictions

```diff
@@ -175,6 +175,7 @@
     {
       "SelectorRegexp": "k8s[.]io/kubernetes/pkg",
       "AllowedPrefixes": [
+        "k8s.io/kubernetes/pkg/api/pod",
```

Comment:
was this required? the only imports this PR adds are to `k8s.io/kubernetes/pkg/api/v1/pod` which was already allowed

Reply:
This looks like a leftover import from cleanup. Will refactor this and update

Reply:
still outstanding